src/App.jsx

```diff
@@ -0,0 +1,28 @@
+import './App.css'
+import { useState, useEffect } from 'react';
+
+function App() {
+  const [success, setSuccess] = useState('');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const getMessage = await fetch('http://localhost:3000/message');
+      const res = await getMessage.json();
+      setSuccess(res);
+    };
+    fetchData();
+  }, []);
+
+  return (
+    <div className="App">
+      <h2>setup RoR + React project as two apps</h2>
+      <div className="card">
+        <h3 className="read-the-docs">
+          " {success.message} " This message is displayed from backend API
+        </h3>
+      </div>
+    </div>
```

Comment:
- [ ] Great job so far, however according to the project requirements, You should Create a `Greeting` component that will display a greeting and Set it up as a route in your App component. Please address this 👍🏻
 
- [ ]  As I said above 🙃  It will be great for the `App` component to have a route that leads to the `Greeting` component. This is a major requirement asked in the project description for this project to be approved. ⬇️ 


- [ ] There should be a store, action and reducer that uses the url to the Rails API endpoint (as nothing is deployed here, it can be a placeholder) Actually you missed the redux part, please add this 👇🏻 Here are the project requirements

![Screenshot from 2023-01-03 03-31-44](https://user-images.githubusercontent.com/79058364/210289974-c72f4748-1371-4647-93eb-885b397f96af.png)

Reply:
@BenMukebo Thank you for pointing out some of the issues that were missing from the requirement, I have updated my code to the requirement specifications, Thank you for your depth feedback really appreciate it.